Soccar [UI]/src/nl/socnet/message/EventMessage.java

```diff
@@ -0,0 +1,48 @@
+package nl.socnet.message;
+
+import nl.soccar.library.enumeration.EventType;
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+import nl.socnet.message.handler.EventMessageHandler;
+
+/**
+ * Message for occurring events.
+ *
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.EVENT_MESSAGE, handler = EventMessageHandler.class)
+public final class EventMessage extends Message {
+
+    private final EventType type;
+    private final int playerId;
+
+    /**
+     * Initializes this message.
+     *
+     * @param playerId The id of the new host.
+     */
+    public EventMessage(EventType type, int playerId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): type [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)